projects/storefrontlib/src/lib/product/components/product-list/product-view/product-view.component.scss

```diff
@@ -1,143 +1,97 @@
-$cx-product-search-layout-grid-mobile-trbl-padding: 13px 12px 13px 12px !default;
-
-$cx-product-search-layout-grid-span-before-background-color: 'inverse' !default;
-$cx-product-search-layout-grid-span-before-height: 2px !default;
-$cx-product-search-layout-grid-span-before-width: 100% !default;
-$cx-product-search-layout-grid-span-before-top: 50% !default;
-$cx-product-search-layout-grid-span-before-left: 0 !default;
-$cx-product-search-layout-grid-span-before-trbl-margin: -1px 0 0 0 !default;
-
-$cx-product-search-dropdown-trbl-padding: 0 35px 0 0 !default;
-
-$cx-product-search-layout-list-trbl-padding: 13px 18px 13px 18px !default;
-$cx-product-search-layout-list-mobile-trbl-padding: 13px 12px 13px 12px !default;
-$cx-product-search-layout-list-span-color: 'secondary' !default;
-$cx-product-search-layout-list-span-hover-color: 'primary' !default;
-
-$cx-product-search-layout-list-before-font-size: 25px !default;
-$cx-product-search-layout-list-before-bottom: 12px !default;
-
-.cx-product-search__layout {
-  position: relative;
-  display: inline-block;
-  overflow: hidden;
-  border-radius: $cx-product-search-layout-border-radius;
-  border: $cx-product-search-layout-border;
-  @include var-color('border-color', $cx-product-search-layout-border-color);
-  max-height: $cx-product-search-layout-height;
+.cx-product-layout {
+  position: var(--cx-position, relative);
+  display: var(--cx-display, inline-block);
+  overflow: var(--cx-overflow, hidden);
+  border-radius: var(--cx-border-radius, 4px);
+  border-width: var(--cx-border-width, 1px);
+  border-style: var(--cx-border-style, solid);
+  border-color: var(--cx-border-color, var(--cx-g-color-light));
+  max-height: var(--cx-max-height, 48px);
 
   span {
-    position: relative;
-    display: inline-block;
-    width: $cx-product-search-layout-span-width;
-    height: $cx-product-search-layout-span-height;
-    overflow: hidden;
+    position: var(--cx-position, relative);
+    display: var(--cx-display, inline-block);
+    overflow: var(--cx-overflow, hidden);
+    width: var(--cx-width, 20px);
+    height: var(--cx-height, 20px);
   }
+}
 
-  &--grid {
-    display: inline-block;
-    padding: $cx-product-search-layout-grid-trbl-padding;
-    @include var-color('border-color', $cx-product-search-layout-border-color);
+.cx-product-grid {
+  display: var(--cx-display, inline-block);
+  padding: var(--cx-padding, 13px 18px 13px 18px);
+  color: var(--cx-g-color-light);
 
-    @include media-breakpoint-down(sm) {
-      padding: $cx-product-search-layout-grid-mobile-trbl-padding;
-    }
+  @include media-breakpoint-down(sm) {
+    padding: var(--cx-padding, 13px 12px 13px 12px);
+  }
 
-    span {
-      @include var-color('color', $cx-product-search-layout-color);
-      @include var-color(
-        'background',
-        $cx-product-search-layout-background-color
-      );
+  span {
+    color: var(--cx-g-color-inverse);
```

Comment:
This should be `color: var(--cx-color, --cx-g-color-inverse);` & `background-color: var(--cx-background-color, --cx-g-color-secondary);`